### ISSUE TITLE
--reverse-proxy not working as expected when running kubedock as a standalone service

### ISSUE THREAD

**Issue by rcgeorge23** [joyrex2001/kubedock#18]
I have kubedock running as a standalone service in namespace A. 

I am running a test that uses kubedock to spin up a MySQL pod in namespace B.

I can see that the MySQL pod has started up. When I exec into a pod in namespace A I can successfully nc MySQL using the pod IP (on port 3306) and cluster IP (on the randomly assigned port).

However I can't connect to the kubedock pod using its pod IP and the randomly assigned port MySQL service port.

I am running kubedock with the following arguments:
```
      args:
        - "server"
        - "--image-pull-secrets=xxxx"
        - "--namespace=B"
        - "--reverse-proxy"
```

Would you expect kubedock to work when run like this? Is there anything obvious I'm doing wrong?

**Comment by joyrex2001** [comment]
Could it be blocked due to network policies on your cluster? I don't see another immediate reason why it wouldn't connect to the cluster ip.

**Comment by rcgeorge23** [comment]
Hiya, ok so it seems like from inside the kubedock pod, I can hit the reverse proxy just fine.

I think the problem is that the proxy is only listening on localhost, so it cannot be accessed from outside of the pod.

Is it possible to tell kubedock to bind the proxy to another IP, like the pod IP, or maybe 0.0.0.0?

**Comment by joyrex2001** [comment]
Good find! Are you able to change it and do some tests?

**Comment by rcgeorge23** [comment]
Yeah I’m going to try!

**Comment by rcgeorge23** [comment]
Yep, just switching out `localhost` for `0.0.0.0` has done the trick.
Do you think this should be made configurable, or would it be ok to just change it to `0.0.0.0`?

**Comment by joyrex2001** [comment]
I am ok switching it to 0.0.0.0. Can you make a pull request?

**Comment by rcgeorge23** [comment]
https://github.com/joyrex2001/kubedock/pull/19

**Comment by joyrex2001** [comment]
Thanks, merged!
